### ISSUE TITLE
Wireguard stops working after a few minutes

### ISSUE THREAD

**Issue by andreimc** [StreisandEffect/streisand#1522]
### Expected behavior:

It should not silently disconnect, it should keep working. It breaks the whole connection.

### Actual Behavior:

It stops working.
<img width="671" alt="screen shot 2019-02-28 at 11 32 57 am" src="https://user-images.githubusercontent.com/1446134/53532878-c169cb00-3b4c-11e9-96cc-c4aac9a4d86d.png">

### Steps to Reproduce:
1. Fresh install
2. Connect wireguard on mac using: https://itunes.apple.com/au/app/wireguard/id1451685025?mt=12
3. ping www.google.com

[ contents of `streisand-diagnostics.md` here ]

**Comment by yutaowang** [comment]
PersistentKeepalive = 25

??

**Comment by andreimc** [comment]
Yeah still does the same thing, I tried several wireguard servers all on digitalocean tho.
